app/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"html/template"
+	"math"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/domainr/whois"
+	flags "github.com/jessevdk/go-flags"
+
+	"github.com/akellbl4/WoD/config"
+)
+
+var opts struct {
+	Config     string `long:"config" short:"c" env:"WOD_CONFIG" description:"Path to config.json" default:"./config.json"`
+	Dest     string `long:"destination" short:"d" env:"WOD_DESTINATION" description:"Path to destination directory" default:"./web"`
+	Template string `long:"template" short:"t" env:"WOD_TEMPLATE_PATH" description:"Path to html template" default:"./template.html"`
+}
+
+var TemplateData struct {
+	Domain          string
+	Price           int
+	RegistratedDate string
+}
+
+func main() {
+	if _, err := flags.Parse(&opts); err != nil {
+		os.Exit(1)
+	}
+
+	cfg := config.ReadConfg(otps.Config)
```

Comment:
Just FYI, right now it seems to contain typos in both function name and options reference. Also I would suggest renaming opts.Config to ConfigPath, so it would look like `cfg := config.Parse(opts.ConfigPath)` which is more clear in my opinion.